Documentation/Getting-Started/Prerequisites/prerequisites.md

```diff
@@ -101,3 +101,50 @@ or choose a different Linux distribution.
 If you will be creating volumes from a Ceph shared file system (CephFS), the recommended minimum kernel version is **4.17**.
 If you have a kernel version less than 4.17, the requested PVC sizes will not be enforced. Storage quotas will only be
 enforced on newer kernels.
+
+## Distro Notes
+
+Specific configurations for some distributions.
+
+### NixOS
+
+When you use NixOS, the kernel modules will be found in the non-standard path `/run/current-system/kernel-modules/lib/modules/`,
+and they'll be symlinks to stuff located inside the also non-standard path `/nix`.
+
+For Rook Ceph containers to be able to load the required modules, they need read access to those locations.
+
+If you install Rook with Helm, add this to your `values.yaml`:
```

Comment:
How about if we leave the complex example settings out of this doc and just mention the setting names?
```suggestion
If you install Rook with Helm, uncomment the examples settings in `values.yaml`:
- `csiCephFSPluginVolume`
- `csiCephFSPluginVolumeMount`
- `csiRBDPluginVolume`
- `csiRBDPluginVolumeMount`
```